packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -562,6 +562,26 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   @override
   void jumpTo(double value);
 
+  @override
+  Future<void> moveTo(double to, {
+    Duration duration,
+    Curve curve,
+    bool clamp = true,
+  }) {
+    assert(to != null);
+    assert(clamp != null);
+
+    if (clamp)
+      to = to.clamp(minScrollExtent, maxScrollExtent);
+
+    if (duration == null || duration == Duration.zero) {
```

Comment:
could the next few lines be replaced by `super.moveTo(...)`?

Reply:
Good point, yes.